pkg/pdp/service/roots_add.go

```diff
@@ -383,105 +367,27 @@ func (p *PDPService) AddRoots(ctx context.Context, id uint64, request []types.Ro
 			return common.Hash{}, fmt.Errorf("computing height and padding: %w", err)
 		}
 		// NB: defined here: https://github.com/FilOzone/pdp/blob/main/src/PDPVerifier.sol#L44
-		// as MAX_PIECE_SIZE_LOG2 = 50
```

Comment:
Removed hard coded values, and rely on the value the contract abi expects instead. removes a "magic" number.